R/delete.R

```diff
@@ -32,7 +32,7 @@
 #' docdb_delete(src, "mtcars")
 #'
 #' # mongo
-#' src <- src_mongo("stuff")
+#' src <- src_mongo(collection = "iris")
```

Comment:
The last line in this mongo example now fails

```r
src <- src_mongo(collection = "iris")
docdb_create(src, "iris", iris)
docdb_get(src, "iris")
docdb_delete(src)
#> Error in docdb_delete.src_mongo(src) :
#>  argument "key" is missing, with no default
```

Reply:
Thanks, completed and corrected examples in documentation with commit 19b6d9b.